src/commands/commit/index.js

```diff
@@ -21,8 +22,14 @@ const promptAndCommit = (options: CommitOptions): Function =>
   getEmojis()
     .then((gitmojis) => prompts(gitmojis, options))
     .then((questions) => {
-      inquirer.prompt(questions).then((answers) => {
-        if (options.mode === COMMIT_MODES.HOOK) return withHook(answers)
+      inquirer.prompt(questions).then((answers: Answers) => {
+        // Make sure the title is capitalized
+        answers.title =
+          answers.title.charAt(0).toUpperCase() + answers.title.slice(1)
```

Comment:
Let's avoid mutating the `answers` object here:

```suggestion
   const transformedAnswers = {
     ...answers,
     title: answers.title.charAt(0).toUpperCase() + answers.title.slice(1)
   }
```